test/test_memory.cpp

```diff
@@ -197,19 +286,20 @@ TEST_F(TMATest, LoadCompleteTensor1D) {
 
   tv1->axis(0)->parallelize(ParallelType::Bulk);
 
-  auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
-  auto t0 = at::randn({32}, options);
+  auto options =
+      at::TensorOptions().dtype(data_type_to_aten(dtype)).device(at::kCUDA, 0);
+  auto t0 = at::randn({innerDimSize()}, options);
   FusionExecutor fe;
-  fe.compileFusion(&fusion, {t0}, {}, {DataType::Int32});
+  fe.compileFusion(&fusion, {t0}, {}, matmul_cparams);
```

Comment:
The `matmul_cparams` is to make sure the `nvfuser_zero` is disabled, so smem allocation is well aligned.